Backend/dev/migrations/rider-app/1105-add-place-name-cache-table.sql

```diff
@@ -0,0 +1,10 @@
+CREATE TABLE IF NOT EXISTS atlas_app.place_name_cache(
+  id character(36) NOT NULL PRIMARY KEY,
+  formatted_address TEXT,
+  plus_code TEXT,
+  lat DOUBLE PRECISION,
+  lon DOUBLE PRECISION,
+  place_id varchar (255),
+  address_components Text[],
+  geo_hash TEXT
+);
```

Comment:
`lat`, `lon`, `addressComponents` should be NOT NULL as it follows from your tabular code

Reply:
ok

Reply:
resolve this too